main.py

```diff
@@ -1,11 +1,50 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+
+    # Перевірка структури даних
+    print(players_data)  # Для налагодження можна вивести дані
+
+    for player in players_data:
+        if isinstance(player, dict):  # Перевіряємо, чи це словник
+            race_name = player.get("race", {}).get("name")
+            race_desc = player.get("race", {}).get("description")
+            race, created = Race.objects.get_or_create(
+                name=race_name,
+                description=race_desc
+            )
+
+            # Створюємо навички для раси, якщо вони є
+            skills_data = player.get("race", {}).get("skills", [])
+            for skill in skills_data:
+                Skill.objects.get_or_create(
+                    name=skill["name"],
+                    bonus=skill["bonus"]
+                )
```

Comment:
When creating a `Skill`, you should associate it with a `Race`. The current code does not set the `race` field for the `Skill` model, which is required as per the model definition. You need to pass the `race` instance to the `get_or_create` method for `Skill`.